docusaurus/docs/iOS/components/reactions.md

```diff
@@ -45,110 +34,178 @@ let customReactions: [MessageReactionType: ChatMessageReactionAppearanceType] =
 ```
 
-### ChatMessageReactionsView
+In the example above we changed the color of the bubble to black. For this we had not only to change the background to black, but also the border color and the tail images. We also changed the corner radius of the bubble since in the next step we will make the bubble vertical instead of horizontal.
+
+### ChatReactionPickerReactionsView
 
-This component shows the list of reactions, you can customize this if you want to change how single reactions are positioned or arranged.
+This component is resposinble to layout the list of reactions. By default the reactions are rendered in a horizontal stack, exactly like the message reaction view.
 
+Let's do the same change we did with the message reaction view, and change the stack to be vertical aligned:
 ```swift
-class CustomChatChatMessageReactionsView: ChatMessageReactionsView {}
+class CustomChatReactionPickerReactionsView: ChatReactionPickerReactionsView {
 
-Components.default.messageReactionsView = CustomChatChatMessageReactionsView.self
+    override func setUpLayout() {
+        super.setUpLayout()
+
+        stackView.axis = .vertical
+    }
+}
+
+Components.default.reactionPickerReactionsView = ChatReactionPickerReactionsView.self
 ```
 
 ### ChatMessageReactionItemView
 
-This component renders the single message reaction. 
+The component responsible to display a single reaction is actually the same as the one used by the message reaction view. Let's reuse the same customization we did the message reaction view and apply it to the reaction picker too:
 
 ```swift
 class CustomChatMessageReactionItemView: ChatMessageReactionItemView {
-    override open func updateContent() {
-        super.updateContent()
 
-        imageView?.layer.borderWidth = 1.0
-        imageView?.layer.cornerRadius = 3.0
-        imageView?.layer.masksToBounds = true
-        imageView?.layer.borderColor = .init(red: 0, green: 0, blue: 0, alpha: 1)
+    override var reactionImageTint: UIColor? {
+        guard let content = content else { return nil }
+        let isChosenByCurrentUser = content.reaction.isChosenByCurrentUser
+        return isChosenByCurrentUser ? .white : .gray
     }
 }
 
-Components.default.messageReactionItemView = CustomChatMessageReactionItemView.self
+Components.default.reactionPickerReactionItemView = CustomChatMessageReactionItemView.self
 ```
 
-| Default Message Reactions | Custom Message Reactions |
-| ------------------------- | ------------------------ |
-| ![Default](../assets/default-message-reaction.png)  | ![Custom](../assets/custom-message-reaction.png)  |
+Here is the result for all the customizations done to the reaction picker view:
+
+| Default Picker | Custom Picker |
+| ------------- | ------------- |
+| ![Default Picker](../assets/message-reactions-tap.png)  | ![Custom Picker](../assets/message-reactions-picker-custom.png)  |
+
+## Reaction Authors
+
+TODO
```

Comment:
Only this part of the documentation re-write that is missing. After that the PR is 100% complete